modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMDataDefinitionConverterImpl.java

```diff
@@ -272,19 +272,24 @@ private DDMFormField _createFieldSetDDMFormField(
 				setLabel(
 					new LocalizedValue() {
 						{
-							addString(defaultLocale, StringPool.BLANK);
+							for (Locale locale : availableLocales) {
+								addString(
+									locale,
+									_language.get(locale, "fields-group"));
+							}
 						}
 					});
-				setLocalizable(false);
+				setLocalizable(true);
 				setNestedDDMFormFields(nestedDDMFormFields);
 				setProperty("collapsible", false);
 				setProperty("ddmStructureId", ddmStructureId);
 				setProperty("ddmStructureLayoutId", ddmStructureLayoutId);
+				setProperty("labelAtStructureLevel", true);
 				setProperty("upgradedStructure", upgradedStructure);
 				setReadOnly(false);
 				setRepeatable(repeatable);
 				setRequired(false);
-				setShowLabel(false);
+				setShowLabel(true);
```

Comment:
@Mikellorza I believe the reported failures [here](https://github.com/liferay-forms/liferay-portal/pull/2216#issuecomment-1544098218) ([DDMDataDefinitionConverterTest](https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/485659/testReport/com.liferay.dynamic.data.mapping.util.test/DDMDataDefinitionConverterTest)) may be related to these changes.

Would you mind taking a look?

Reply:
fixed, thanks